### PR TITLE
Fix grammatical and syntax errors;

### PR DESCRIPTION
**Pull request by laney0808**
Fix grammar and syntax problems mentioned in #75. 

Additionally, improve consistency in the use of '`Class`s' plural by using `Class`s instead of `Classes` througout; 
Improve consistency in using colon after a "as follows" or "let's..." sentence;

### CODE REVIEW

File: tutorials/javaFxPart1.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 </frontmatter>
 
 # JavaFX tutorial part 1 – Introduction
+<div style="color:gray; margin:0 0 20px 0; font-size:10px">Last updated: 13/8/2024 </div>
```
Comment (damithc) [inline tutorials/javaFxPart1.md:7]: We can remove 'last updated' from all five pages, as they are a pain to maintain and doesn't really add value.

File: tutorials/javaFxPart1.md [hunk 2]
```diff
@@ -172,11 +173,12 @@ Congratulations! You have created your first GUI application!
...
-   1. Can you have more than one `Stage` an application?
+   1. Can you have more than one `Stage` in an application?
    1. Try creating another stage and showing it! What happens?
 
 [:fas-arrow-up: **ToC**](javaFx.md) | <span class="badge rounded-pill bg-primary">**++What's next?++**</span> [:fas-arrow-right: JavaFX tutorial part 2 - **Creating a GUI for Duke**](javaFxPart2.md)
 
 --------------------------------------------------------------------------------
 **Authors:**
 * Initial Version: Jeffry Lum
+* Contributors: Zhang Lanyu
```
Comment (damithc) [inline tutorials/javaFxPart1.md:184]: We add name here only for significant contributions.

File: tutorials/javaFxPart5.md [hunk 3]
```diff
@@ -346,7 +346,7 @@ To **view the effects of the margin**, you may either
 
 ## Borders
 
-Labels, TextBoxes, Buttons, more or less all the nodes used so far are subclasses of [Region](https://docs.oracle.com/javafx/2/api/javafx/scene/doc-files/cssref.html#region), and hence share the following CSS properties for borders:
+`Label`s, `TextBoxe`s, `Button`s, more or less all the nodes used so far are subclasses of [Region](https://docs.oracle.com/javafx/2/api/javafx/scene/doc-files/cssref.html#region), and hence share the following CSS properties for borders:
```
Comment (damithc) [inline tutorials/javaFxPart5.md:349]:
```suggestion
`Label`s, `TextBox`es, `Button`s, more or less all the nodes used so far are subclasses of [Region](https://docs.oracle.com/javafx/2/api/javafx/scene/doc-files/cssref.html#region), and hence share the following CSS properties for borders:
```

**Review by damithc — CHANGES_REQUESTED** [review]
Thanks for the PR @laney0808 
Some minor changes suggested.

File: tutorials/javaFxPart2.md
Comment (jedkohjk) [inline tutorials/javaFxPart2.md:1]: Can't add comments on lines not edited. Line 40:

> From the diagram, you see that the root `Node` can contain many other `Nodes` and similarly, each of those `Nodes` can contain many other `Nodes`. This means that if we can find a _container_ to set as our root `Node`, we can place all our other `Nodes` in it.

If we standardised to `Node`s in part 1, should we do the same here?

File: tutorials/javaFxPart4.md
Comment (jedkohjk) [inline tutorials/javaFxPart4.md:1]: Can't comment on lines not edited.

Line 146: DialogBox -> `DialogBox`

Can consider using a colon after lines 174, 178 and 184

Line 293 says Scene Builder. Line 295 says SceneBuilder. (Majority of part 4 says Scene Builder. Majority of part 5 says SceneBuilder.)

**Review by jedkohjk — COMMENTED** [review]
Overall, LGTM!

File: tutorials/javaFxPart4.md
Comment (laney0808) [reply tutorials/javaFxPart4.md:1]: Thank you for pointing this out. I shall open another PR to fix this by using Scene Builder.

File: tutorials/javaFxPart1.md (hunk 2 above)
Comment (laney0808) [reply tutorials/javaFxPart1.md]: Thank you! Fixed in new commit.

File: tutorials/javaFxPart1.md (hunk 1 above)
Comment (laney0808) [reply tutorials/javaFxPart1.md]: Thank you! Fixed in new commit.

File: tutorials/javaFxPart2.md
Comment (laney0808) [reply tutorials/javaFxPart2.md:1]: Thank you! Fixed in new commit.

File: tutorials/javaFxPart4.md
Comment (laney0808) [reply tutorials/javaFxPart4.md:1]: Scene Builder problem fixed in #77. The other probles fixed in new commits.

**Comment by damithc** [comment]
Thanks for the PR @laney0808 and for the review @jedkohjk 
I'm merging this quickly so that later reviewers don't report the same problems again.

**Pull request merged**